paddlepaddle/paddlepaddle-model-zoo/src/main/java/ai/djl/paddlepaddle/zoo/cv/wordrecognition/PpWordRecognitionTranslator.java

```diff
@@ -84,7 +84,8 @@ public Batchifier getBatchifier() {
             w = 32.0 / min * w;
         }
         int h32 = (int) h / 32;
-        int w32 = (int) w / 32;
-        return new int[] {h32 * 32, w32 * 32};
+        int w32 = Math.max((int) w / 32 / h32, 1);
+        // height must be 32 (fixed)
+        return new int[] {32, w32 * 32};
```

Comment:
if height is fixed to 32 shouldn't the algorithm be just w = 32 / h * w ?

Reply:
We still need to make sure width are base 32 number and has proper aspect ratio

Reply:
Does this work:

int ratio = Math.floorDiv(w, h);

Reply:
No, this doesn't apply here.

Reply:
floorDiv cannot be used on floating number

Reply:
As Jake pointed out, if you height is always 32, the width should simply be 32 * ratio,
no need to test if size is less then 32 or not. the whole function can be:
```
int ratio =  Math.max((int) w / h, 1);
return new int[] {32, 32 * ratio};
```

Reply:
I tried this way, the last inference result is always wrong. Will try this implementAction again.